locales/en.default.schema.json

```diff
@@ -2166,7 +2166,7 @@
         },
         "show_rating": {
           "label": "Show product rating",
-          "info": "To display a rating, add a product rating app. [Learn more](https://help.shopify.com/manual/online-store/themes/os20/themes-by-shopify/dawn/page-types#product-recommendations-show-product-rating)"
+          "info": "To display a rating, add a product rating app. [Learn more](https://help.shopify.com/en/manual/online-store/themes/theme-structure/page-types#product-recommendations-section-settings)"
         },
         "header_mobile": {
```

Comment:
I also see an old url for `featured-collection`'s `show_rating` but there is redirect working that doesn't lead to a 404 page: https://help.shopify.com/manual/online-store/themes/os20/themes-by-shopify/dawn/sections#featured-collection-show-product-rating

As well as for `featured-product`'s `rating` using: https://help.shopify.com/manual/online-store/themes/os20/themes-by-shopify/dawn/sections#featured-product-rating

Reply:
I think those would be the last 2 using a reference to `os20` in their url.

Reply:
Good catch! I'll update them!